src/ts/components/seekbar.ts

```diff
@@ -80,6 +88,12 @@ export class SeekBar extends Component<SeekBarConfig> {
 
   private player: PlayerAPI;
 
+  protected seekBarType: SeekBarType;
+
+  protected getSeekBarType = () => {
```

Comment:
Why an arrow function? And why above the constructor?

Reply:
The scope of the class gets lost when the method is passed to the controller. I can either do it as arrow function or do `getSeekBarType.bind(this)`. Not sure which one is preferred

Reply:
Please convert to a conventional class function and pass as `() => this.getSeekBarType()` instead of `this.getSeekBarType`